src/redux/actions.ts

```diff
@@ -0,0 +1,233 @@
+import { Language } from '../component/LanguageSelector';
```

Comment:
i followed the convention of creating low-level actions that are then dispatched by the action creators in the `actionCreators.ts`